.claude/behavior/principles.md

```
@@ -1,5 +1,15 @@
 # Behavioral Principles & Working Style
 
+ ## Library API Verification (MANDATORY)
```

markdownlint-cli2 (0.18.1)

Headings must start at the beginning of the line

(MD023, heading-start-left)